core/domain/state_domain.py

```diff
@@ -772,65 +772,75 @@ def validate(self):
                 self.needs_update)
 
 
-class ContentTranslations(object):
+class WrittenTranslations(object):
     """Value object representing a content translations which stores
     translated contents of all state contents (like hints, feedback etc.) in
     different languages linked through their content_id.
     """
 
-    def __init__(self, content_translations):
-        """Initializes a ContentTranslations domain object."""
-        self.content_translations = content_translations
+    def __init__(self, translations_mapping):
+        """Initializes a WrittenTranslations domain object."""
+        self.translations_mapping = translations_mapping
 
     def to_dict(self):
-        """Returns a dict representing this ContentTranslations domain object.
+        """Returns a dict representing this WrittenTranslations domain object.
 
         Returns:
-            dict. A dict, mapping all fields of ContentTranslations instance.
+            dict. A dict, mapping all fields of WrittenTranslations instance.
```

Comment:
If this is the case, shouldn't there be a 'translations_mapping' key in the dict?

Ideally your object and the corresponding dict would "look" the same.

Reply:
I have changed the doctstring!